docs/ros/ros/tutorials/driving.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Driving a robot
+sidebar_label: Teleop
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+:::caution
+Make sure you have successfully [set up](../installation/robot) the robot before attempting to drive the robot.
+:::
+
+There are several ways to drive a Clearpath robot over ROS 2.
+
+### Joystick Teleoperation
+
+Make sure your controller is paired to the robot PC. Visit the [controller pairing](../installation/controller) guide for details.
+
+With the [robot services](../installation/robot#robot-services) running, turn on your controller. Press and hold either **L1** or **R1**, and use
+the left joystick to publish a linear velocity and yaw command. The **L1** button will drive the robot at 'normal' speeds, while **R1** will drive the robot at 'turbo' speeds.
```

Comment:
We'll need to update this when we get include Dingo-O and Ridgeback, right? Should we also include some images here? We should have something from the ROS1 tutorials that we can use.